docs/contributing/submit-to-starter-library.md

```diff
@@ -1,55 +1,59 @@
 
 ```yaml:title=docs/starters.yml
-- url: Link to a demo of your starter
-  repo: Link to GitHub repo
-  description: Your starter description
+- url: Link para uma demonstração de seu starter
+  repo: Link para o repositório do GitHub
+  description: Sua descrição do starter
 
-  # These correspond to the category filters in the library
-  # See docs/categories.yml for valid tags.
+  # Estes correspondem aos filtros na biblioteca
+  # Veja docs/categories.yml tags válidas.
   tags:
     - Redux
 
-  # Add your site features
-  # These will be included on your starter's detail page.
+  # Adicione funcionalidades ao seu site
+  # Elas serão incluídas na página de detalhes do seu starter.
   features:
-    - Blog post listing with previews (image + summary) for each blog post
+    - Lista de postagens do blog com visualizações (imagem + resumo) de cada postagem do blog
 ```
 
-Use the following template to ensure required fields are filled:
+Use o seguinte modelo para garantir que os campos obrigatórios sejam preenchidos:
 
 ```yaml:title=docs/starters.yml
-- url: (required)
-  repo: (required - https://github.com/{username}/{titleofthesite})
-  description: (optional)
+- url: (obrigatório)
+  repo: (obrigatório - https://github.com/{usuario}/{titulodosite})
+  description: (opcional)
   tags:
-    - (required)
+    - (obrigatório)
   features:
-    - (required)
+    - (obrigatório)
 ```
 
-Check out the [`starters.yml`](https://github.com/gatsbyjs/gatsby/blob/master/docs/starters.yml) file for examples.
+Confira o arquivo [`starters.yml`](https://github.com/gatsbyjs/gatsby/blob/master/docs/starters.yml) para exemplos.
 
-We prefer pull requests with the title in the following format:
-`chore(starters): add my-starter-name-here`
-If there are linting issues with your PR, you can fix them by running `npm run format`.
+Preferimos solicitações pull com o título no seguinte formato:
+`chore (iniciantes): adicione meu nome de iniciante-aqui`
+Se houver problemas com o seu PR, você poderá corrigi-los executando o `npm run format`.
 
-### Need to change details?
+Preferimos pull requests com o título no seguinte formato:
+`chore(starters): adicionar nome-do-meu-starter-aqui`
+Se houver problemas com o seu PR, você poderá corrigi-los executando o `npm run format`.
 
-If you want to edit anything in your site submission later, simply edit the .yml file by submitting another PR. GitHub data (like stars) will be automatically pulled and updated, but your starter description, tags, and feature list are up to you!
+### Precisa alterar detalhes?
 
-### Adding new tag
+Se você deseja editar qualquer coisa no envio do seu site posteriormente, basta editar o arquivo .yml enviando outro PR. Os dados do GitHub (como estrelas) serão automaticamente puxados e atualizados, mas sua descrição, tags e lista de funcionalidades são de sua responsabilidade!
```

Comment:
edit the .yml file by submitting another PR -> Submeta uma nova PR modificando/atualizando o arquivo .yml.
(Nao recomendo seguir a mesma estrutura do ingles nesse caso pois a interpretacao vai ficar bem dificil)